src/core_ocean/shared/mpas_ocn_vmix_cvmix.F

```diff
@@ -530,25 +534,37 @@ subroutine ocn_vmix_coefs_cvmix_build(meshPool, statePool, forcingPool, diagnost
               ! Build deltaVelocitySquared
               do i = 1, nEdgesOnCell(iCell)
                  iEdge = edgesOnCell(i, iCell)
+                 cell1 = cellsOnEdge(1, iEdge)
+                 cell2 = cellsOnEdge(2, iEdge)
+
                  deltaVelocitySquared(1) = 0.0_RKIND
 
-                 normalVelocitySum(1) = normalVelocity(1, iEdge)
+                 layerThicknessEdge = 0.5_RKIND*(layerThickness(1, cell1) + layerThickness(1, cell2))
```

Comment:
We have a layerThicknessEdge variable in the Registry. Of course this is a little calculation, but `layerThicknessEdge(iEdge)` has a better memory access for an edge loop. I can change that.

Reply:
Thanks for catching that.  Not sure why I didn't think of that array!  Thanks for making the change too